community/lucene-index/src/test/java/org/neo4j/index/recovery/UniqueIndexRecoveryTests.java

```diff
@@ -92,7 +94,15 @@ public void before()
         extensionFactories.add( kernelExtensionFactory );
         extensionFactories.add(new LuceneLabelScanStoreExtension());
         factory.setKernelExtensions( extensionFactories );
-        db = (GraphDatabaseAPI) factory.newEmbeddedDatabase( storeDir.absolutePath() );
+        db = newDb();
+    }
+
+    private GraphDatabaseAPI newDb()
+    {
+        return (GraphDatabaseAPI) factory
+                .newEmbeddedDatabaseBuilder( storeDir.absolutePath() )
+                .setConfig( GraphDatabaseSettings.label_index, LabelIndex.LUCENE.name() )
+                .newGraphDatabase();
```

Comment:
That's really a test of recovery of indexes and uniqueness constraints. Adding a native variant wouldn't really add anything. It's however unclear why the lucene label index is used specifically.